episodes/4-minimal-reproducible-code.Rmd

```diff
@@ -128,6 +128,14 @@ counts_per_day <- krats %>%
   group_by(year, plot_id, plot_type, month, day, species_id) %>%
   summarize(count_per_day = n())
 
+### ***EDIT-start *** ###
+
+# --> Why doesn't this use "date" like we had set up earlier?
```

Comment:
Good point--I don't remember when we set up `date`, and I should definitely go back and use that. Will keep in mind this continuity point for future.